typescript/src/entity.ts

```diff
@@ -535,6 +535,16 @@ export class Entity {
     }
   }
 
+  public merge = (otherEntity: Entity, onlyAttributes?: Array<string>) => {
+    const mergeProperty = (info: PropertyInfo) => {
+      const value = (otherEntity as any)[info.attribute];
+      if (value !== undefined && (!onlyAttributes || onlyAttributes.includes(info.attribute))) {
+        (this as any)[info.attribute] = value;
+      }
+    };
+    this.forEachProperty(mergeProperty);
+  }
```

Comment:
note that this doesn't preserve attribute cleanliness information. that may or may not  be what you want.

Reply:
if it is what you want, you ought to add that to the test case